src/app/log.py

```diff
@@ -26,13 +27,43 @@ def configure_logging():
                 "class": "logging.StreamHandler",
                 "formatter": "mozlog_json",
                 "stream": sys.stdout,
-            }
+            },
+            "null": {
+                "class": "logging.NullHandler",
+            },
         },
         "loggers": {
-            "request.summary": {"handlers": ["console"], "level": "INFO"},
-            "backoff": {"handlers": ["console"], "level": "INFO"},
-            "src.jbi": {"handlers": ["console"], "level": "DEBUG"},
+            "request.summary": {
+                "handlers": ["console"],
+                "level": logging.INFO,
+                "propagate": False,
```

Comment:
What will `"propagate": False,`  do here?

Reply:
It will prevent the log messages emitted with as `request.summary` to be caught by loggers called `"request"` or `""` (=`root`)